pages/hub/apps/guides/advanced/live/errors.mdx

```diff
@@ -91,6 +91,16 @@ Increase the `relayerFeeAmount` (notifying the user about it) value and resend t
 The bet transaction may fail for some reason. Usually we get the fail code and include it in the `errorMessage` field.
 Usually the best option here is just to resend the request.
 
+### `OrderError.TooLowRelayerFeeAmount`
+
+This error means that the order amount is too small and the cost of gas required for the transaction is greater than the order amount. 
+Please increase the order amount.
```

Comment:
The error appears to be related to a low relayer fee amount rather than the order amount

Reply:
fixed